### PR TITLE
add bsc archive node snapshot

### PR DESCRIPTION
**Pull request by pythonberg1997**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,17 +4,18 @@ This repository provides snapshot files for the [Reth](https://github.com/bnb-ch
...
+| Network       | Type        | Frequency | Size (Compressed) | Size (Decompressed) | Download Link                                                                                     | CheckSum                                                                 |
+|---------------|-------------|-----------|-------------------|---------------------|---------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------|
+| BSC Mainnet   | FullNode    | Monthly   | 2.7TB             | 3.81TB              | [Download](https://pub-c0627345c16f47ab858c9469133073a8.r2.dev/reth-20240819.tar.lz4)             | bbb325df85ed4cb9ee4c65c4102fc12d(MD5)                                    |
+| BSC Mainnet   | ArchiveNode | Monthly   | 2.7TB             | 3.81TB              | [Download](https://pub-c0627345c16f47ab858c9469133073a8.r2.dev/bsc-reth-archive-20240910.tar.lz4) | pending                                                                  |
```
Comment (forcodedancing) [inline README.md]: Size is not correct `2.7TB | 3.81TB`

File: guild/bsc-reth-snapshot.md [hunk 2]
```diff
@@ -46,3 +48,13 @@ bsc-reth node \
     --http.api="eth, net, txpool, web3, rpc" \
     --full
 ```
+
+### ArchiveNode
+
```
Comment (unclezoro) [inline guild/bsc-reth-snapshot.md:53]: why fullnode and archive node share same start command? They should be different

File: README.md (hunk 1 above)
Comment (pythonberg1997) [reply README.md]: Fixed

File: guild/bsc-reth-snapshot.md (hunk 2 above)
Comment (pythonberg1997) [reply guild/bsc-reth-snapshot.md:53]: There's no `--full` for archive node

**Review by forcodedancing — APPROVED** [review]

**Review by unclezoro — APPROVED** [review]

**Pull request merged**